lib/renderer/api/remote.js

```diff
@@ -281,6 +282,16 @@ function handleMessage (channel, handler) {
   })
 }
 
+const enableStacks = hasSwitch('enable-api-filtering-logging')
+
+function getCurrentStack () {
+  const target = {}
+  if (enableStacks) {
+    Error.captureStackTrace(target)
```

Comment:
TIL `Error.captureStackTrace`. What a weird API! If I'm reading https://v8.dev/docs/stack-trace-api right, it looks like we could make this stack a little cleaner by passing `getCurrentStack` as the 2nd argument to `captureStackTrace`, which would exclude that function from the trace?

Reply:
@nornagon that would be great. I will check it out

Reply:
@nornagon I've actually passed the `remote` method instead of `getCurrentStack` as the 2nd argument, as the log message already says which remote method was called.